.github/workflows/entropy-check.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: entropy-check
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Would this workflow ever need to run on a push to main (outside a PR)? If not, consider removing this trigger.